### DISCUSSION TITLE
Trying to show the time in the transient prompt

### DISCUSSION THREAD

**Discussion by SvenAelterman** [JanDeDobbeleer/oh-my-posh#4094] [Troubleshoot] [answered]
I'd like the transient prompt to show the time of when the command started execution.

I've first tried to get the output of my `rprompt`'s time segment. That output shows as blank. When I remove the `.Output` property, it shows what looks like the template of the segment. But in any case, that would be the time the prior command execution completed, so it would not give me what I am after anyway.

Is this possible?

**Comment by SvenAelterman** [comment]
PS: I've tried to add this to my transient prompt template:

`{{ .CurrentDate | date .Format }}`

but it results in this error message:

`: nil pointer evaluating template.Data.CurrentDate`

And none of the transient prompt shows.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@SvenAelterman you can make use of `{{ date }}` and the formatting opting listed in [here](http://masterminds.github.io/sprig/date.html). For example `now | date "2006-01-02"`

**Reply by SvenAelterman** [reply]
Thanks. If anyone wants the answer (because date formatting in Go is a little unexpected), here's what I used (in JSON):

`{{now | date \"15:04:05\"}}`

**Reply by joaociocca** [reply]
thanks for sharing <3 I also wanted to use date, instead of just time, so in the end I also had to check the weird formatting used by Go... and in that case, you do `{{now | date \"2006-01-02 15:04:05\"}}`
